morfo_segmentace_výpočet_malu_tokens.py

```diff
@@ -27,57 +27,54 @@ def vypocet_mal(data):
 # nastavení "lokality"
 setlocale(LC_NUMERIC, "cs_CZ.UTF-8")
 
-cisla = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "slouč"]
-
-for cislo in cisla:
-
-    # načtení segmentovaného textu
-    with open(f"vysledek_segmentace_stud_{cislo}.txt", encoding="UTF-8") as soubor:
-        segmentovany_text_tokens = soubor.read().strip().split(sep=" ")
-
-    # přípravné výpočty
-    # výpočet délky konstruktu v konstituentech
-    delka_slov_v_morfech = []
-    for slovo in segmentovany_text_tokens:
-        delka_slova_v_morfech = slovo.count("-") + 1
-        delka_slov_v_morfech.append(delka_slova_v_morfech)
-
-    # výpočet délky konstruktu v subkonstituentech
-    nesegmentovany_text = []
-    for segmentovane_slovo in segmentovany_text_tokens:
-        hole_slovo = segmentovane_slovo.replace("-", "")
-        nesegmentovany_text.append(hole_slovo)
-
-    delka_slov_ve_fonemech = []
-    for slovo in nesegmentovany_text:
-        delka_slova_ve_fonemech = len(slovo)
-        delka_slov_ve_fonemech.append(delka_slova_ve_fonemech)
-
-    # počet x-konstituentových konstruktů
-    frekvence_morfu = Counter(delka_slov_v_morfech)
-
-    # slovník: klíč = x-konstituentový konstrukt, hodnota = součet délek všech takových konstruktů (dvou-morfémové slovo, součet délek všech dvou-morfémových slov)
-    soucty_delek_x_morfovych_slov = {}
-    for i, klic in enumerate(delka_slov_v_morfech):
-        if klic not in soucty_delek_x_morfovych_slov:
-            soucty_delek_x_morfovych_slov[klic] = 0
-        soucty_delek_x_morfovych_slov[klic] += delka_slov_ve_fonemech[i]
-
-    print(soucty_delek_x_morfovych_slov)
-
-    # slovník: klíč = x-konstituentový konstrukt, hodnota = (součet délek všech takových konstruktů, počet takových konstuktů)
-    slovnik_data_pro_mal = {}
-    for klic in soucty_delek_x_morfovych_slov:
-        slovnik_data_pro_mal[klic] = (soucty_delek_x_morfovych_slov[klic], frekvence_morfu[klic])
-
-    print(dict(sorted(slovnik_data_pro_mal.items())))  # seřazený slovník podle klíčů, pozor na seřazování slovníku - ošemetné, pro zobrazení či tahání infa ale stačí
-
-    vysledek_mal = vypocet_mal(slovnik_data_pro_mal)
-    print(vysledek_mal)
-
-    # uložení výsledků do tabulky
-    with open(f"data_S_M_F_tokens_stud_{cislo}.csv", "x", encoding="UTF-8") as csvfile:
-        vysledek_data = csv.writer(csvfile, delimiter=';', lineterminator='\n')
-        vysledek_data.writerow(["construct", "frq", "mean of constituent"])
-        for i in vysledek_mal:
-            vysledek_data.writerow([i[0], i[1], i[2]])
+# načtení segmentovaného textu
+with open("vysledek_segmentace_stud_.txt", encoding="UTF-8") as soubor:
```

Comment:
Musí tam být to poslední podtržítko? V mnou navrhovaných změnách nebylo. V souboru [_morfo_segmentace_automatická.py_](https://github.com/pelegrinova/morfo_segmentace/pull/12/files#diff-ed55999b2c9d7b0a0ecce79cd91eeac0df2bdeba28f31a1d4c2bc645f41a8190) je též nemáš.

```suggestion
with open("vysledek_segmentace_stud.txt", encoding="UTF-8") as soubor:
```

Reply:
Nemusí, nějak mi to přišlo jedno, protože je to tak jako tak dementně pojmenované, resp. očekávám v brzké době, že obsah těhle řetězců stejně budeme řešit nějak obecněji.

Reply:
Souhlasím, ale čistě technicky je to chyba při řešení konfliktu. To by totiž mělo být pouze čistým sloučením změn bez zanášení nových.

Reply:
Souhlas.

Reply:
Opraveno v #18.